config/application.rb

```diff
@@ -239,7 +239,8 @@ class Application < Rails::Application
       :telemetry => {:enabled => false},
       :blob => {:enabled => false},
       :taxonomy => {:enabled => true},
-      :api_deprecations => {:enabled => true}
+      :api_deprecations => {:enabled => true},
+      :sidekiq => {:enabled => true, :level => :warn}
```

Comment:
Interesting that Rubocop doesn't suggest a trailing comma here

Reply:
Should I take this you're taking rubocop's role and suggesting that?

Reply:
It looks like this doesn't happen for other similar calls so leave it.